testworld/signature_test.go

```diff
@@ -0,0 +1,61 @@
+// +build testworld
+
+package testworld
+
+import (
+	"github.com/centrifuge/centrifuge-protobufs/gen/go/coredocument"
+	"github.com/centrifuge/go-centrifuge/documents"
+	"github.com/centrifuge/go-centrifuge/identity"
+	"github.com/centrifuge/go-centrifuge/testingutils/config"
+	"github.com/centrifuge/go-centrifuge/testingutils/documents"
+	"github.com/centrifuge/go-centrifuge/utils"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestHost_Signature(t *testing.T) {
+	t.Parallel()
+	bob := doctorFord.getHostTestSuite(t, "Bob")
+	eve := doctorFord.getHostTestSuite(t, "Eve")
+
+	ectxh := testingconfig.CreateAccountContext(t, eve.host.config)
+
+	collaborators := [][]byte{bob.id[:]}
+	coredoc := prepareCoreDocument(t, collaborators, eve)
+
+	err := eve.host.p2pClient.GetSignaturesForDocument(ectxh, coredoc)
+	assert.NoError(t, err)
+	assert.Equal(t, 2, len(coredoc.Document.Signatures))
```

Comment:
lets make this fail :), Eve is not going to sign with her own signature anyway ;)

Reply:
@vimukthi-git -- So the test is failing but the error is in the `p2p` layer and is not being returned by `p2pClient.GetSignaturesForDocument`. This method always returns `nil` even though there's an error. Any work arounds you can think of? Thanks!

Reply:
@eswarasai @vedhavyas would help you with that, please contact him on slack.